bin/strata-dbtool/src/cmd/client_state.rs

```diff
@@ -30,10 +30,33 @@ pub(crate) fn get_client_state_update(
     args: GetClientStateUpdateArgs,
 ) -> Result<(), DisplayedError> {
     let block_id = parse_l1_block_id(&args.block_id)?;
-    let block_mf = get_l1_block_manifest(db, block_id)?;
-    let block_commitment: L1BlockCommitment = block_mf
-        .ok_or(DisplayedError::InternalError("".to_string(), Box::new(())))?
-        .into();
+    let block_mf = get_l1_block_manifest(db, block_id)?.ok_or(DisplayedError::InternalError(
+        "Block manifest not found".to_string(),
+        Box::new(()),
+    ))?;
+
+    // Find the height for this block ID by searching the canonical chain
+    let (tip_height, _) = get_l1_chain_tip(db)?;
+    let height = (0..=tip_height)
```

Comment:
I'm not so happy with this, but can we at least traverse from (up to, actually) the Strata genesis block?